diff --git a/crates/ui/src/main.rs b/crates/ui/src/main.rs
index 716c45e..2700ccf 100644
--- a/crates/ui/src/main.rs
+++ b/crates/ui/src/main.rs
@@ -19,7 +19,7 @@ fn main() {
                         .with_title("Scrabble Desktop")
                         .with_resizable(true)
                         .with_min_inner_size(dioxus_desktop::tao::dpi::LogicalSize::new(800, 600))
-                        .with_inner_size(dioxus_desktop::tao::dpi::LogicalSize::new(1400, 900)),
+                        .with_inner_size(dioxus_desktop::tao::dpi::LogicalSize::new(1400, 1150)),
                 ),
             )],
         );
